draft-irtf-cfrg-frost.md

```diff
@@ -888,8 +888,8 @@ in this document assumes the following threat model.
 the protocol, although participants still are able to verify the consistency of their
 shares via a VSS (verifiable secret sharing) step; see {{dep-vss}}.
 
-* Unforgeability assuming less than `(t-1)` corrupted signers. So long as an adverary
-corrupts fewer than `(t-1)` participants, the scheme remains secure against EUF-CMA attacks.
+* Unforgeability assuming at most `(t-1)` corrupted signers. So long as an adverary
+corrupts fewer than `t` participants, the scheme remains secure against EUF-CMA attacks.
```

Comment:
Can we also drop a citation to EUF-CMA security while we're at it?

Reply:
Is there one you had in mind? Otherwise I'll just cite Boneh/Shoup, I guess.